src/test/kotlin/uk/gov/dluhc/printapi/rds/mapper/CertificateToPrintRequestMapperTest.kt

```diff
@@ -0,0 +1,167 @@
+        // Given
+        val requestId: String = aValidRequestId()
+        val sourceReference: String = aValidSourceReference()
+        val applicationReference: String = aValidApplicationReference()
+        val vacNumber: String = aValidVacNumber()
+        val vacVersion = "1"
+        val sourceType: SourceType = SourceType.VOTER_CARD
+        val requestDateTime: Instant = Instant.ofEpochMilli(0)
+        val firstName = "John"
+        val middleNames = "Anthony Barry"
+        val surname = "Doe"
+        val certificateLanguage = CertificateLanguage.EN
+        val certificateFormat = CertificateFormat.STANDARD
+        val delivery = rdsDeliveryBuilder()
+        val gssCode: String = getRandomGssCode()
+        val issuingAuthority: String = aValidLocalAuthorityName()
+        val issueDate = LocalDate.of(2022, 10, 21)
+        val suggestedExpiryDate = LocalDate.of(2032, 10, 21)
+        val eroEnglish: ElectoralRegistrationOffice = rdsElectoralRegistrationOfficeBuilder(name = issuingAuthority)
+        val photoLocation = aPhotoArn()
+        val statusHistory = mutableListOf(
+            PrintRequestStatus(
+                status = Status.PENDING_ASSIGNMENT_TO_BATCH,
+                eventDateTime = Instant.now()
+            )
+        )
+        val batchId = aValidBatchId()
+        val printRequest = PrintRequest(
+            requestId = requestId,
+            vacVersion = vacVersion,
+            requestDateTime = requestDateTime,
+            firstName = firstName,
+            middleNames = middleNames,
+            surname = surname,
+            certificateLanguage = certificateLanguage,
+            certificateFormat = certificateFormat,
+            photoLocationArn = photoLocation,
+            delivery = delivery,
+            eroEnglish = eroEnglish,
+            eroWelsh = eroWelsh,
+            batchId = batchId,
+            userId = aValidUserId(),
+            statusHistory = statusHistory
+        )
+        val certificate = Certificate(
+            vacNumber = vacNumber,
+            sourceType = sourceType,
+            sourceReference = sourceReference,
+            applicationReference = applicationReference,
+            applicationReceivedDateTime = Instant.now(),
+            issuingAuthority = issuingAuthority,
+            issueDate = issueDate,
+            suggestedExpiryDate = suggestedExpiryDate,
+            status = Status.PENDING_ASSIGNMENT_TO_BATCH,
+            gssCode = gssCode,
+            printRequests = mutableListOf(printRequest)
+        )
+        val photoZipPath: String = aPhotoZipPath()
+
+        val expected = uk.gov.dluhc.printapi.printprovider.models.PrintRequest()
+        expected.requestId = requestId
+        expected.issuingAuthorityEn = eroEnglish.name
+        expected.issueDate = LocalDate.parse("2022-10-21")
+        expected.suggestedExpiryDate = LocalDate.parse("2032-10-21")
```

Comment:
Curious as why can't you set these expected dates to issueDate and suggestedExpiryDate?

Reply:
Don't think there is one now - I think elsewhere had issues with Offset v Instant... updated.